video/vaapi.c

```diff
@@ -166,8 +180,8 @@ bool va_guess_if_emulated(struct mp_vaapi_ctx *ctx)
 }
 
 struct va_native_display {
-    void (*create)(VADisplay **out_display, void **out_native_ctx,
-                   const char *path);
+    void (*create)(struct mpv_global *global, VADisplay **out_display,
```

Comment:
if just for the sake of logging why not pass only `struct mp_log *log`?

Reply:
Generally modules are passed as first argument, no?